docs/references/contact-self.md

```diff
@@ -2,120 +2,156 @@
 title: ContactSelf
 ---
 
-Bot itself will be encapsulated as a ContactSelf. This class is extends Contact.
+机器人本身被封装为一个`ContactSelf`类。 这个类继承自联系人`Contact`类。
 
 ## ContactSelf
 
-Bot itself will be encapsulated as a ContactSelf.
+机器人本身被封装为一个`ContactSelf`类。
 
-> Tips: this class is extends Contact
+> 提示: 这个类继承自联系人`Contact`类。
 
-**Kind**: global class
+**类型**: 公共类
 
-* [ContactSelf](contact-self.md#contactself)
+* [contact_self](contact-self.md#contactself)
   * [intance](contact-self.md#contactself)
-    * [contactSelf.avatar\(\[file\]\) ⇒ `Promise <void | FileBox>`](contact-self.md#contactselfavatarfile-⇒-promise)
-    * [contactSelf.qrcode\(\) ⇒ `Promise <string>`](contact-self.md#contactselfqrcode-⇒-promise)
-    * [contactSelf.signature\(signature\) ⇒ `Promise <string>`](contact-self.md#contactselfsignaturesignature)
-    * [contactSelf.name\(\[name\]\) ⇒ `Promise <void> | string`](contact-self.md#contactselfname-⇒-promisestring)
+    * [contact_self.avatar\(\[file\]\) ⇒ `FileBox`](contact-self.md#contactselfavatarfile-⇒-promise)
+    * [contact_self.qrcode\(\) ⇒ `str`](contact-self.md#contactselfqrcode-⇒-promise)
+    * [contact_self.signature\(signature\) ⇒ `Any`](contact-self.md#contactselfsignaturesignature)
+    * [contact_self.name\(\[name\]\) ⇒ `str | None`](contact-self.md#contactselfname-⇒-promisestring)
```

Comment:
Btw, the reason why I put the `str | None` here is because the an getter setter method
The source code is here
```python
    @property
    def name(self) -> str:
        """
        :return:
        """
        return super().name

    @name.setter
    def name(self, name: Optional[str]) -> None:
        puppet_id: str = self.puppet.self_id()

        if self.contact_id != puppet_id:
            raise WechatyOperationError('only can get qr_code for the login user self')

        asyncio.run(self.puppet.contact_self_name(name))
```
should I change it to `Optional[str]`, or `Optional[str] | None`